can_tools/scrapers/official/CA/ca_state.py

```diff
@@ -5,20 +5,39 @@
 import requests
 import us
 
-from ... import DatasetBaseNoDate
-from ..base import CMU, StateDashboard
+from can_tools.scrapers.base import DatasetBaseNoDate, CMU
+from can_tools.scrapers.official.base import StateDashboard
 
 
 class OpenDataCali(StateDashboard, ABC):
```

Comment:
If we end up writing other scrapers for California, we may end up moving this to a file like `CA/common.py` or something like that -- Nothing to do for now, but just a comment that this may be useful in other contexts.